ui/src/app/shared/action/action.component.ts

```diff
@@ -88,8 +83,8 @@ export class ActionComponent implements OnDestroy, OnInit {
     }
 
     ngOnInit() {
-        this.actionSub = this._actionStore.getProjectActions(this.project.key).subscribe(mapActions => {
-            this.publicActions = mapActions.valueSeq().toArray().filter((action) => action.name !== this.editableAction.name);
+        this._actionService.getAllForProject(this.project.key).subscribe(as => {
```

Comment:
Is it relevant to put a loader here ?

Reply:
Will be nice but this list is used by a child component, so we'll have to do some changes also in children.